plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js

```diff
@@ -8,17 +8,24 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import TimerImage from './timer.svg';
-import PersonalizeMyStore from './illustrations/personalize-my-store.js';
```

Comment:
Should we delete the [illustrations folder](https://github.com/woocommerce/woocommerce/tree/370601c3d49f734c3f12eab08bf12c20e53e2b34/plugins/woocommerce-admin/client/two-column-tasks/headers/illustrations)?

Reply:
Yeah, I originally left it, incase we were going to revert, but that is what Git is for... so deleting it makes the most sense.

Reply:
Removed here: https://github.com/woocommerce/woocommerce/pull/32805/commits/711e0ce44935ed54586f91119779846f4a563508